WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -13,6 +13,12 @@ extension UIImage {
         return UIImage.gridicon(.addOutline)
     }
 
+    /// Arrow Up Icon
```

Comment:
I would not consider this a blocker, but as far as I recall, we used to add a test for these icons in `IconsTests` (a very simple test that ensures that the image is not nil). A bit overkill, but prevents a runtime crash if an image is missing from the final binary.

Reply:
Aaaaand I saw the tests down below. Sorry!